cli/cmd/generate.go

```diff
@@ -388,6 +391,60 @@ func addWorkloadOwnerKeyToManifest(manifst *manifest.Manifest, keyPath string) e
 	return nil
 }
 
+type logTranslator struct {
+	r         *io.PipeReader
+	w         *io.PipeWriter
+	logger    *slog.Logger
+	stopDoneC chan struct{}
+}
+
+func newLogTranslator(logger *slog.Logger) logTranslator {
+	r, w := io.Pipe()
+	l := logTranslator{
+		r:         r,
+		w:         w,
+		logger:    logger,
+		stopDoneC: make(chan struct{}),
+	}
+	l.startTranslate()
+	return l
+}
+
+func (l logTranslator) Write(p []byte) (n int, err error) {
+	return l.w.Write(p)
+}
+
+var genpolicyLogPrefixReg = regexp.MustCompile(`^\[[^\]\s]+\s+(\w+)\s+([^\]\s]+)\] (.*)`)
```

Comment:
Could you please add a test for this regexp? That would also serve to document what it's supposed to capture.